src/lib.rs

```diff
@@ -78,20 +82,15 @@ impl From<checked_command::Error> for Error {
 }
 
 impl error::Error for Error {
-    fn description(&self) -> &str {
-        "Process error"
-    }
-
-    fn cause(&self) -> Option<&error::Error> {
-        Some(self)
-    }
+    fn description(&self) -> &str { "Process error" }
+    fn cause(&self) -> Option<&error::Error> {Some(self)}
```

Comment:
This is the rustfmt style so it is not "mine".

Reply:
Sounds good to me. I don't use rustfmt, so I wouldn't know, but neither way is wrong. This is mostly a preference thing.